.github/workflows/workflow.yml

```diff
@@ -70,11 +70,11 @@ jobs:
           pip install pipenv
           pip install tox
       - name: Integration tests
-        run: tox -e py3.7 -- -m integration
+        run: tox -e py3.7 -- -rfE -m integration
 
   platform_checks:
     runs-on: ${{ matrix.os }}
-
+    continue-on-error: true
```

Comment:
I think we should remove this again to avoid burning too many minutes.

Reply:
removed. used only for testing.